src/gf.c

```diff
@@ -134,8 +134,8 @@ static int speccache_eq(size_t idx, const void *ty, jl_svec_t *data, uint_t hv)
 // get or create the MethodInstance for a specialization
 static jl_method_instance_t *jl_specializations_get_linfo_(jl_method_t *m JL_PROPAGATES_ROOT, jl_value_t *type, jl_svec_t *sparams, jl_method_instance_t *mi_insert)
 {
-    if (m->sig == (jl_value_t*)jl_anytuple_type && jl_atomic_load_relaxed(&m->unspecialized) != NULL)
-        return jl_atomic_load_relaxed(&m->unspecialized); // handle builtin methods
+    //if (m->sig == (jl_value_t*)jl_anytuple_type && jl_atomic_load_relaxed(&m->unspecialized) != NULL)
+    //    return jl_atomic_load_relaxed(&m->unspecialized); // handle builtin methods
```

Comment:
When do we ever actually reach this for other builtins?

Reply:
I thought I have seen some generic code paths reach there, since there is nothing that would stop a user from requesting it when they attempt introspection (e.g. PrecompileUtils), and resulting in a polluted cache here.

Reply:
Ok, I'll have this check for the OpaqueClosure method then.